src/include/homestore/checkpoint/cp.hpp

```diff
@@ -70,6 +70,11 @@ class CPContext;
 class CPManager;
 
 VENUM(cp_consumer_t, uint8_t,
+      // Sealer is a special consumer that provides information regarding where the cp is up to.
+      // It will be the first one during cp switch over , as a conservative marker of everything
+      // before or equals to this point, should be in current cp, possibly some consumer are above this point which is fine.
+      // And Sealer is the last one during cp flush after all other services flushed successfully.
+      SEALER = 3,
```

Comment:
SEAKER has the same emum number as REPLICATION_SVC, both of them are 3 ?

Reply:
it is not made for dynamic registration which I dont think we have the use case atm.